data_safe_haven/administration/users/entra_id_users.py

```diff
@@ -11,8 +11,8 @@
 from .research_user import ResearchUser
 
 
-class AzureADUsers:
-    """Interact with users in an Azure Active Directory"""
+class EntraIDUsers:
+    """Interact with users in an Entra ID directory."""
```

Comment:
I think this is supposed to just be `Entra ID` rather than `Entra ID directory` i.e. `Entra ID` replaces `Azure Active Directory`. I find it cumbersome not to have directory there, personally, but that's the official guidance

Reply:
:+1:

We discussed this in a previous PR https://github.com/alan-turing-institute/data-safe-haven/pull/1665#issuecomment-1959661195

Luckily Microsoft has [documentation](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad) on renaming Azure AD.

Reply:
Interact with users in an ID? OK Microsoft.

Reply:
> Interact with users in an ID? OK Microsoft.

*Exactly* - it's **so** awkward